source/includes/steps-start-sharded-cluster.yaml

```diff
@@ -0,0 +1,48 @@
+title: "Start config servers."
+stepnum: 1
+ref: start-config-servers
+content: |
+
+   Start the :binary:`~bin.mongod` instance for each of the
+   :doc:`config servers </core/sharded-cluster-config-servers>`, specifying
+   ``--configsvr``, ``--replSet``, ``--bind_ip``, and other options as
+   appropriate to your deployment.
+
+   .. code-block:: shell
+
+      mongod --configsvr --replSet <replicaSetName> --bind_ip localhost,<hostname(s)|ip address(es)>
+
+---
+title: "Start shards."
+stepnum: 2
+ref: start-shards
+content: |
+
+   Start the :binary:`~bin.mongod` instance for each replica set member of
+   each :doc:`shard </core/sharded-cluster-shards>`, adding options as
+   appropriate for your deployment.
+
+---
+title: "Start :binary:`mongos <bin.mongos>` routers."
+stepnum: 3
+ref: start-mongos
+content: |
+
+   Start each :binary:`mongos <bin.mongos>` router, adding options as
+   appropriate for your deployment:
+
+   .. code-block:: shell
+
+      mongos --config /path/to/config/mongos.conf
+
+   If the CSRS replica set name or any member hostname changed, update the
+   :binary:`mongos <bin.mongos>` configuration file setting
+   :setting:`sharding.configDB` with updated configuration server
+   connection string:
+
+   .. code-block:: yaml
+
+      sharding:
+        configDB: "myNewCSRSName/config1.example.net:27019,config2.example.net:27019,config3.example.net:27019"
+
```

Comment:
Assuming the balancer was stopped prior to shutting down the sharded clusters, I think should have a step to restart or verify the balancer state. Also potentially testing that all shards are up and healthy.

Reply:
done